portlayer/exec/container.go

```diff
@@ -176,3 +176,23 @@ func (c *Container) Start(ctx context.Context) error {
 
 	return nil
 }
+
+func (c *Container) Stop(ctx context.Context) error {
+	defer trace.End(trace.Begin("Container.Stop"))
+
+	if c.vm == nil {
+		return fmt.Errorf("vm not set")
+	}
+
+	//TODO: make the shutdown much cleaner, right now we just pull the plug on the vm.(may need corresponding work in tether.)
+
+	// Power off
+	_, err := tasks.WaitForResult(ctx, func(ctx context.Context) (tasks.ResultWaiter, error) {
```

Comment:
It looks like govmomi already covers this case. same for start if you try to start a container that is already started then you will get a change state error.